### PR TITLE
Add a page "Staying up to date"

### PR DESCRIPTION
**Pull request by aibolik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,3 +45,5 @@ Front-End FAQ will be the one place you can go to for all your Front-end questio
 [Observables](observables.md)
 
 [Testing](testing.md)
+
+[Staying up to date](to-follow.md)
```
Comment (emmabostian) [inline README.md]:
```suggestion
[Staying up to date](staying-up-to-date.md)
```

File: to-follow.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Staying up to date
```
Comment (emmabostian) [inline to-follow.md]: Please re-name this file to `staying-up-to-date.md`

File: to-follow.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+<details>
+<summary>JavaScript and FrontEnd</summary>
+
+> This is an opiniated list of influencers and other people/companies who are really good at what they are doing. They all relate somehow to FrontEnd and/or JavaScript and most of them constantly share knowledge through blog posts and/or twitter discussions. Feel free to contribute to this list.
```
Comment (emmabostian) [inline to-follow.md]:
```suggestion
> This is an opiniated list of influencers and other people/companies who are really good at what they are doing. They all relate somehow to FrontEnd and/or JavaScript and most of them constantly share knowledge through blog posts and/or Twitter discussions. Feel free to contribute to this list.
```

File: to-follow.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Web components are a complete waste of time and effort at this point. The champions of web components are too invested to admit it, but there are so many problems with the current spec. They haven&#39;t learned anything from the current state of the art of building UI for the web.</p>&mdash; MICHAEL JACKSON (@mjackson) <a href="https://twitter.com/mjackson/status/1050594839127973896?ref_src=twsrc%5Etfw">October 12, 2018</a></blockquote>
+
+
+- Kyle Simpson([@getify](https://twitter.com/getify)) - when we talk about JavaScript it is unfair to not include Kyle Simpson, also knows as GETIFY. My foundation of JavaScript strengthened after reading his [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS) book series. I highly recommend to all new(not only beginners) JS devs. You better go to his twitter account, learn about him and read 2 books mentioned in his profile.
```
Comment (emmabostian) [inline to-follow.md]:
```suggestion
- Kyle Simpson([@getify](https://twitter.com/getify)) - when we talk about JavaScript it is unfair to not include Kyle Simpson, also known as GETIFY. My foundation of JavaScript strengthened after reading his [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS) book series. I highly recommend to all new JS devs (not only beginners) . You should go to his Twitter account, learn about him, and read the two books mentioned in his profile.
```

**Review by emmabostian — CHANGES_REQUESTED** [review]
This is so great thanks! Just a couple of minor comments. Additionally you should add Sarah Drasner!

**Comment by aibolik** [comment]
@emmawedekind sure, thanks for feedback. Will update

**Comment by aibolik** [comment]
@emmawedekind  Can you please review? I've made changes you asked. Thanks

P.S.: I was just not sure if GitHub makes notifications that I made changes.

**Review by emmabostian — APPROVED** [review]
Great thanks!

**Pull request merged**